lib/girouette/test/girouette/garden/util_test.cljc

```diff
@@ -0,0 +1,29 @@
+(ns girouette.garden.util-test
+  (:require [clojure.test :refer [deftest testing is are]]
+            [girouette.tw.default-api :refer [class-name->garden]]
+            [girouette.tw.common :refer [dot]]
+            [girouette.garden.util :refer [apply-class-rules]]))
+
+(deftest apply-class-rules-test
+  (are [class-name classes expected-garden]
+       (= expected-garden
+          (apply-class-rules (mapv class-name->garden classes)
+                                   (dot class-name)
+                                   (set (map dot classes))))
```

Comment:
Small indentation problem here - I will fix it, no need to update to code.

Reply:
Ah, looks like my brain tricked me on this one :)